plugins/generator-1.14.4/forge-1.14.4/procedures/advancement_isequalto.java.ftl

```diff
@@ -0,0 +1,2 @@
+((((MinecraftServer) ((ServerPlayerEntity) entity).server).getAdvancementManager()
+                .getAdvancement(new ResourceLocation("${generator.map(field$achievement, "achievements")}"))) == ((dependencies.get("advancement"))))
```

Comment:
`(dependencies.get("advancement")` - this is not the right way. Define new dependency type advancement in https://github.com/MCreator/MCreator/blob/master/plugins/generator-1.15.2/forge-1.15.2/mappings/types.yaml, https://github.com/MCreator/MCreator/blob/master/plugins/generator-1.14.4/forge-1.14.4/mappings/types.yaml, and https://github.com/MCreator/MCreator/blob/d9ee2c56227c194c3770d70d75531bdb6bb2d392/src/main/java/net/mcreator/blockly/Dependency.java#L64.

Then define new dependency in procedure block JSON similar to how you declare world depedency on others and directly reference `advancement` in the code.

Then update `dependencies_provided` of your trigger with the new dependency.

Reply:
It's done now!